FirebaseCore/Sources/FIRApp.m

```diff
@@ -838,11 +838,25 @@ + (void)registerSwiftComponents {
   SEL componentsToRegisterSEL = @selector(componentsToRegister);
   // Dictionary of class names that conform to `FIRLibrary` and their user agents. These should only
   // be SDKs that are written in Swift but still visible to ObjC.
-  NSDictionary<NSString *, NSString *> *swiftLibs = @{@"FIRFunctionsComponent" : @"fire-fun"};
-  for (NSString *className in swiftLibs.allKeys) {
+  NSDictionary<NSString *, NSString *> *swiftComponents = @{@"FIRFunctionsComponent" : @"fire-fun"};
+  for (NSString *className in swiftComponents.allKeys) {
     Class klass = NSClassFromString(className);
     if (klass && [klass respondsToSelector:componentsToRegisterSEL]) {
-      [FIRApp registerInternalLibrary:klass withName:swiftLibs[className]];
+      [FIRApp registerInternalLibrary:klass withName:swiftComponents[className]];
+    }
+  }
+
+  // Swift libraries that don't need component behaviour
+  NSDictionary<NSString *, NSString *> *swiftLibraries = @{
+    @"FIRCombineAuthComponent" : @"comb-auth",
+    @"FIRCombineFirestoreComponent" : @"comb-firestore",
+    @"FIRCombineFunctionsComponent" : @"comb-functions",
+    @"FIRCombineStorageComponent" : @"comb-storage",
```

Comment:
Given we're using the `component` name for components being registered above, and `libraries` for non-components that we just want to keep track of, what do you think about changing the `Component` suffix with `Library`?